sites/partners/src/lib/listings/formTypes.ts

```diff
@@ -167,6 +167,7 @@ export const formDefaults: FormListing = {
   //     rows: [],
   //   },
   // },
+  applicationLotteryTotals: [],
```

Comment:
nit: unrelated to this PR and I think I introduced it, but can we delete the commented out code above this